packages/app/src/messages/components/messages-input-search/messages-input-search.component.tsx

```diff
@@ -50,6 +52,8 @@ const renderListItem: RenderListItem<Item<Message & Thread>> = ({
   props,
 }) => {
   const messageContent = Boolean(item.data.content)
+  const phoneNumber = useSelector(getPhoneNumberById(item.data.phoneNumberId))
```

Comment:
It might only be a naming issue but GitHub Actions throws an warning about using a hook outside of Component or another hook.